### PR TITLE
feat: upgrade angular version to 10

### PR DESCRIPTION
**Pull request by sguiheux**
1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

File: ui/package.json [hunk 1]
```diff
@@ -81,31 +81,31 @@
...
+    "ngx-moment": "5.0.0",
     "prismjs": "1.21.0",
     "raven-js": "3.27.2",
     "rxjs": "6.5.4",
     "rxjs-compat": "6.3.3",
     "sanitize-html": "1.20.0",
     "spinkit": "1.2.5",
     "string-format-obj": "1.1.1",
-    "tslib": "^1.10.0",
+    "tslib": "^2.0.0",
```
Comment (richardlt) [inline ui/package.json:99]:
```suggestion
    "tslib": "2.0.0",
```

File: ui/package.json [hunk 2]
```diff
@@ -81,31 +81,31 @@
...
     "ts-helpers": "1.1.2",
     "zone.js": "~0.10.2",
     "zxcvbn": "4.4.2"
   },
   "devDependencies": {
     "@angular-builders/custom-webpack": "9.1.0",
-    "@angular-devkit/build-angular": "~0.901.1",
-    "@angular/cli": "9.1.1",
-    "@angular/compiler-cli": "9.1.2",
+    "@angular-devkit/build-angular": "~0.1001.1",
```
Comment (richardlt) [inline ui/package.json:106]:
```suggestion
    "@angular-devkit/build-angular": "0.1001.1",
```

File: ui/package.json [hunk 3]
```diff
@@ -120,18 +120,18 @@
...
     "copy-webpack-plugin": "4.6.0",
     "husky": "3.0.5",
-    "jasmine-core": "3.5.0",
-    "jasmine-spec-reporter": "4.2.1",
-    "karma": "4.4.1",
-    "karma-chrome-launcher": "3.1.0",
-    "karma-coverage-istanbul-reporter": "2.1.0",
-    "karma-jasmine": "3.1.1",
-    "karma-jasmine-html-reporter": "1.4.2",
+    "jasmine-core": "~3.5.0",
```
Comment (richardlt) [inline ui/package.json:123]:
```suggestion
    "jasmine-core": "3.5.0",
```

File: ui/package.json [hunk 4]
```diff
@@ -120,18 +120,18 @@
...
+    "jasmine-spec-reporter": "~5.0.0",
```
Comment (richardlt) [inline ui/package.json:124]:
```suggestion
    "jasmine-spec-reporter": "5.0.0",
```

File: ui/package.json [hunk 5]
```diff
@@ -120,18 +120,18 @@
...
+    "karma": "~5.0.0",
+    "karma-chrome-launcher": "~3.1.0",
```
Comment (richardlt) [inline ui/package.json:126]:
```suggestion
    "karma-chrome-launcher": "3.1.0",
```

File: ui/package.json [hunk 6]
```diff
@@ -120,18 +120,18 @@
...
-    "jasmine-core": "3.5.0",
-    "jasmine-spec-reporter": "4.2.1",
-    "karma": "4.4.1",
-    "karma-chrome-launcher": "3.1.0",
-    "karma-coverage-istanbul-reporter": "2.1.0",
-    "karma-jasmine": "3.1.1",
-    "karma-jasmine-html-reporter": "1.4.2",
+    "jasmine-core": "~3.5.0",
+    "jasmine-spec-reporter": "~5.0.0",
+    "karma": "~5.0.0",
```
Comment (richardlt) [inline ui/package.json:125]:
```suggestion
    "karma": "5.0.0",
```

File: ui/package.json [hunk 7]
```diff
@@ -120,18 +120,18 @@
...
+    "karma-chrome-launcher": "~3.1.0",
+    "karma-coverage-istanbul-reporter": "~3.0.2",
```
Comment (richardlt) [inline ui/package.json:127]:
```suggestion
    "karma-coverage-istanbul-reporter": "3.0.2",
```

File: ui/package.json [hunk 8]
```diff
@@ -120,18 +120,18 @@
...
+    "karma-jasmine": "~4.0.0",
```
Comment (richardlt) [inline ui/package.json:128]:
```suggestion
    "karma-jasmine": "4.0.0",
```

File: ui/package.json [hunk 9]
```diff
@@ -120,18 +120,18 @@
...
+    "karma-jasmine-html-reporter": "^1.5.0",
```
Comment (richardlt) [inline ui/package.json:129]:
```suggestion
    "karma-jasmine-html-reporter": "1.5.0",
```

File: ui/package.json [hunk 10]
```diff
@@ -120,18 +120,18 @@
...
     "karma-junit-reporter": "1.2.0",
     "lint-staged": "git+https://github.com/bnjjj/lint-staged.git#master",
     "ts-node": "8.3.0",
-    "tslint": "6.1.0",
-    "typescript": "3.8.3",
+    "tslint": "~6.1.0",
```
Comment (richardlt) [inline ui/package.json:133]:
```suggestion
    "tslint": "6.1.0",
```

**Review by yesnault — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#14375.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Docs -   
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✘

**Pull request merged**